### PR TITLE
Fixed sidebar and content sizing issue

### PR DESCRIPTION
**Pull request by julianpoy**
Had to do some extra CSS work to get the sidebar to work correctly (other than just set min-height to 100%).

### CODE REVIEW

File: app/styles/main.scss [hunk 1]
```diff
@@ -138,3 +142,14 @@ $include-html-global-classes: false;
   max-height: 300px;
   border-radius: 8px;
 }
+
+.overflow-auto {
+  overflow: auto;
+}
```
Comment (zachlatta) [inline app/styles/main.scss:170]: Can you add newlines after each CSS block to maintain consistency wit the rest of the CSS?

File: app/styles/main.scss [hunk 2]
```diff
@@ -4,6 +4,10 @@ $include-html-global-classes: false;
 @import "foundation/components/global";
 @import url(http://fonts.googleapis.com/css?family=Amaranth:400,700);
 
+body,html {
```
Comment (zachlatta) [inline app/styles/main.scss]: I hate to be nit-picky like this, but I do want to establish a precedent for consistency. Can you add a space after the comma?

**Comment by zachlatta** [comment]
Great work. I added a few comments about CSS styling. After those are fixed, I'll merge.

**Comment by julianpoy** [comment]
No problem. Its nice to have a more strict organization of code and I like it, although it may take me some time to get used to it.
I look forward to the next bug to fix.

**Comment by julianpoy** [comment]
Mouse drivers need updating... touchpad is all wacky.

**Comment by zachlatta** [comment]
Can you make the map changes a separate PR?

**Comment by julianpoy** [comment]
I wanted to but I wasn't sure how to select what commit to include in the pull request, and how to not update this pull request when new commits are made.

**Comment by zachlatta** [comment]
I ended up reverting these changes in 68589fa1fefc2c970c9d374b78763c220d45b624 because it caused the issue below when the sidebar was used. I think our best option, at least for now, might be to just create longer pages and eventually add a proper contact form.

recorded

**Pull request closed without merging**